docs/usage/directives/configurations/shutdown-time.md

```diff
@@ -0,0 +1,12 @@
+# Shutdown time
+
+`ShutdownTime=[ ISO8601_TIME | none ]`
+
+Allows to set a specific time in the global section when the instances should shutdown (only if the system clock is properly sync with an NTP server).
```

Comment:
I think we can ask chatGPT to rewrite this in a more professional way and also we can set more details of what to expect on date adjustments, etc.

Reply:
Done